### ISSUE TITLE
More options for folders such as favorites and likes

### ISSUE THREAD

**Issue by PanderMusubi** [mopidy/mopidy-dirble#9]
Could you please offer an options that will add a folder with all Dirble favorites and Dirble likes? In that way more the user can find streams.

**Comment by adamcik** [comment]
https://dirble.com/developer/ lists the use API as "coming soon", so for now this isn't something we can add. Feel free to reopen once the API support is there and then we can reconsider.

**Comment by PanderMusubi** [comment]
Thanks for your reply but, sorry for this, isn't it better to leave this issue open? In that way we can off-load it from our minds and keep it in scope.

**Comment by adamcik** [comment]
Fair enough, normally we close things like this for Mopidy itself as it's rather demotivating having hundreds of bugs where many of them aren't actionable.
